app/views/search/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :css do %>
+  <%= stylesheet_link_tag "search/index", "data-turbolinks-track": "reload" %>
+<% end %>
+
+<% provide :title, t(".title") %>
+
+<main>
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-3">
+        <%= render "layouts/sidebar" %>
+      </div>
+      <div class="col-lg-9">
+        <div class="board-list">
+          <div class="header">
+            <p>
+              <%= @boards.size %>
+              <%= t ".searchresult" %>
+              <span class="result">
+                "<%= params[:search] %>"
```

Comment:
chỗ này đẻ nguyên params k dc à e

Reply:
em cho params vào value để lúc search xong nó không bị xóa trắng cái input ạ

Reply:
vâng cái params này là em in ra cái mà người dùng nhập vào sau khi tìm kiếm ạ

Reply:
```suggestion
                <%= params[:search] %>
```